Uses of Closure/setUpHandlers.js

```diff
@@ -0,0 +1,9 @@
+var buttons = document.getElementsByTagName('button');
```

Comment:
There is a problem with the console output when clicking buttons, There result only outputs `Third`

https://github.com/user-attachments/assets/19cc5647-15f3-45d4-a45b-4f3e496efcc1

I believe the output should correlate with the button text which should output `Second` if Second button is clicked same goes with other buttons.

Also, Please don't use `var` keyword when declaring variables as it's prone to bugs and it doesn't respect block of scope.

Please see my suggested fix, to solve the issue
```suggestion
const buttons = document.getElementsByTagName('button');
```